projects/angular-showcase/src/app/business/business-examples/links-showcase/links.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'sbb-links-showcase',
+  templateUrl: './links.component.html'
+})
+export class LinksComponent {
```

Comment:
please rename to LinksShowcaseComponent

Reply:
I was told not to do that https://github.com/sbb-design-systems/sbb-angular/pull/271#discussion_r375220260

Reply:
Bad :). We had problems when components were named the same as components from the library, so LinksComponent and LinkComponent are very similar. Generally we are updating all showcase examples and have the plan to rename them in a constant way. I started it by adding Showcase suffix again wherever i touched it. So currently it's in a mixed state :). We can leave it as it is and I will change it later.